#! /bin/sh -e
## 02_add_debian_files_in_manpage.dpatch by  <weasel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the FILES section of the manpage to properly describe the situation on Debian systems.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad tor-0.1.1.5/doc/tor.1.in /tmp/dpep.E9VjWB/tor-0.1.1.5/doc/tor.1.in
--- tor-0.1.1.12/doc/tor.1.in
+++ /tmp/dpep.E9VjWB/tor-0.1.1.12/doc/tor.1.in
@@ -700,9 +700,15 @@
 .TP
 .B @LOCALSTATEDIR@/lib/tor/
 The tor process stores keys and other data here.
+.TP
+.B /var/log/tor/
+The tor server logs to this directory.
+.TP
+.B /var/run/tor/tor.pid
+The PID of the tor (master) process is stored in this file.
 .LP
 .TP
-.B \fIDataDirectory\fP/approved-routers
+.B /var/lib/tor/approved-routers
 Only for naming authoritative directory servers
 (see \fBNamingAuthoritativeDirectory\fP).
 This file lists nickname to identity bindings.  Each line lists a
